src/main/java/com/exasol/projectkeeper/validators/dependencies/ProjectDependencyReader.java

```diff
@@ -40,8 +41,9 @@ public ProjectDependencyReader(final MavenFileModelReader fileModelReader,
      * @return list of dependencies
      */
     public List<ProjectDependency> readDependencies(final File pomFile) {
-        final var model = parsePomFile(pomFile);
-        return getDependenciesIncludingPlugins(model).map(this::getDependenciesLicense).collect(Collectors.toList());
+        final var project = parsePomFile(pomFile);
+        return getDependenciesIncludingPlugins(project.getModel())
```

Comment:
I think it would be more clear to get plugins and dependencies separately and then merge them instead of doing magic inside `getDependenciesIncludingPlugins` method

Reply:
well, that's what the method does. I thought it would be good to move it to a method and give it a name, since it not very readable otherwise.